aspnetcore/fundamentals/logging/index.md

```diff
@@ -624,6 +624,17 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 You can create your own custom providers as well, to support other logging frameworks or your own logging requirements.
 
+## Azure Log Streaming
+
+Azure log streaming enables you to view log activity in real time from both the application and web server, as well as failed request tracing. Navigate to the **Diagnostics Logs** page from your application's portal page, turn the **Application Logging (Filesystem)** on. 
+
+![Azure portal diagnostic logs page](index/_static/azure-diagnostic-logs.png)
+
+You can then view messages emitted your application through the `ILogger` interface by navigating to the **Log Streaming** page from your application's page in the portal. 
```

Comment:
When Azure log streaming is enabled,  you can view messages by navigating to the **Log Streaming** page from your application's page in the portal.  `[Now tie in `ILogger` interface]` - maybe something like  ALS logs messages through the `ILogger` interface

We need to keep sentences short so the machine translator can do a readable translation.